x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -216,6 +260,10 @@ Similar sequence, just ending with `yarn cypress:open:ccs`, can be used for inte
 
 Appending `--browser firefox` to the `yarn cypress:run:ccs` command above will run the tests on Firefox instead of Chrome.
 
+## Debugging your test
+In order to be able to debug any Cypress test you need to open Cypress on visual mode. [Here](https://docs.cypress.io/guides/guides/debugging)
+you can find an extended guide about how to proceed.
+
```

Comment:
I would remove the empty `## Folder Structure` if we're not doing anything with it right now.

Reply:
I think it's good to point people to the `debugging` section from Cypress, but one good additional tips or tricks might be around when we do experience flake and how we trouble shoot and fix them locally.

One good tip I frequently use is when I see flake I insert a `cy.wait(<some long milliseconds>)` around async parts of our code base such as network calls which can make an indeterministic test, deterministically fail locally for the developer.

The other tip I would add is around how it is very common that we have click handlers not registered and we have to do Cypress pipe and point them to this article about it:
https://www.cypress.io/blog/2019/01/22/when-can-the-test-click/